CHANGELOG.md

```diff
@@ -1,26 +1,36 @@
 # Changelog
 
+## 0.4.0 - Add NMS App Delivery Manager module support (Unreleased)
+
+FEATURES:
+
+- Add the ability to optionally install NMS App Delivery Manager module.
```

Comment:
Did you pull this out too just to get CI in a working state?

Reply:
Yep! Thanks Jason, I'll take that out.